sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsModelData.java

```diff
@@ -62,19 +62,19 @@
      * @param dtdlModel The contents of the model.
      * @param displayName The language map of the localized display names.
      * @param description The language map of the localized descriptions.
-     * @param uploadTime The time when this model was uploaded.
+     * @param uploadedOn The time when this model was uploaded.
      * @param decommissioned If this model has been decommissioned.
      */
     public DigitalTwinsModelData(String modelId,
                                  String dtdlModel,
                                  Map<String, String> displayName,
                                  Map<String, String> description,
-                                 OffsetDateTime uploadTime,
+                                 OffsetDateTime uploadedOn,
                                  boolean decommissioned) {
         this.displayName = displayName;
         this.description = description;
         this.id = modelId;
-        this.uploadTime = uploadTime;
+        this.uploadTime = uploadedOn;
```

Comment:
Would be better to match the property name to uploadedOn as well.

Reply:
This code is mostly copied from a generated type, and the generated type uses uploadTime. In order to keep the manual edits after copying minimal, I'd rather not